100Change2020/ansible/100Change2020.yml

```diff
@@ -16,4 +16,19 @@
   become: true
   become_user: "{{ deployment_user }}"
   roles:
-  - mediawiki
+    - mediawiki
+    - activitylog
+    - collection
+    - embed_video
+    - permissions
+    - simplefavorites
+    # PickSome has to come after collection so the left menu bar is correct
```

Comment:
I can add something to our Collection config that will error if PickSome is already installed.  I'll put it on my todo list, because I don't think it needs to happen in this PR.